### ISSUE TITLE
Finetuning custom dataset

### ISSUE THREAD

**Issue by HelyModi** [open-mmlab/mmdetection#5072]
Hello all,

My aim is to detect cells and tables from images. I have tried several experiments including changing the type of models, the learning-rate, augmentation techniques, however, the models get only around 50% AP for detecting the cells (medium objects) and about 75% AP for detecting the tables (large objects).

Any guidance or suggestion to fine-tuning my model will be appreciated. Thank you!

P.S. I am using cascade mask RCNN right now to build the model.

**Comment by jshilong** [comment]
Sorry for the late response.
You may analyze the scale and aspect ratio of objects and adjust anchors, besides it, augmentation is an effective way to boost the performance.

**Comment by jshilong** [comment]
Feel free to reopen the issue if there is any question
